src/calculation_results.rs

```diff
@@ -130,6 +140,26 @@ impl Factorial {
                     factorial_string, number, digits
                 )
             }
+            CalculatedFactorial::ApproximateDigitsTower(depth, exponent) => {
+                let mut s = if self.is_too_long() || force_shorten {
+                    Self::truncate(exponent, false)
+                } else {
+                    exponent.to_string()
+                };
+                let number = if self.value > *TOO_BIG_NUMBER || force_shorten {
+                    Self::truncate(&self.value, false)
+                } else {
+                    self.value.to_string()
+                };
+                for _ in 0..*depth {
+                    s = format!("10^({s})");
```

Comment:
If the output looks to cluttered to you I could do this:
```suggestion
                for i in 0..*depth {
                    if i == 0 || i == depth.saturating_sub(1) {
                        s = format!("10^({s})");
                    } else {
                        s = format!("10^{s}");
                    }
```
Then it would look like 10 and in superscript 10^10^10^(1938492)